### DISCUSSION TITLE
What's VR architecture and other process methods?

### DISCUSSION THREAD

**Discussion by starshine360** [Anjok07/ultimatevocalremovergui#491] [Q&A]
Since I'm new to this tool, now I'm confused about which kind of model is the best to extract vocals from an audio file. And I don't know what those process methods refer, such as VR architecture and MDX-Net. Would anyone give me answers? Thanks!

**Comment by Binny1977** [comment]
I am new too. I tried different options on the same song and compared them by listening to each. Can anyone please guide where to find the documentation on what different process methods and models do and how they are best suited for different kinds of songs.

**Reply by billiejean1809** [reply]
Scroll down on the github main page. You can also check this link. 
https://youtu.be/ykOKwz3eRUQ
